### PR TITLE
fix(schemas): field name in sie db data type should be camel-case

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Run alteration failed:
image

Inspect the data from the database:
image

## Testing

Alteration succeed:
image

### CODE REVIEW

**Review by gao-sun — APPROVED** [review]

File: packages/schemas/alterations/next-1665300135-sign-in-sign-up.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ const alteration: AlterationScript = {
 
     /* eslint-disable @silverhand/fp/no-mutating-methods */
     if (data) {
-      const { sign_in_methods: signInMethods } = data;
+      const { signInMethods } = data;
```
Comment (gao-sun) [inline packages/schemas/alterations/next-1665300135-sign-in-sign-up.ts:107]: should this be a method with unit tests? cc @simeng-li @wangsijie

Comment (wangsijie) [reply packages/schemas/alterations/next-1665300135-sign-in-sign-up.ts:107]: No, I don't think it is easy to test

**Review by wangsijie — CHANGES_REQUESTED** [review]
The original code works good, would you please do a double check?
image

**Comment by wangsijie** [comment]
btw, I print out the "data":

image

**Comment by wangsijie** [comment]
`pnpm dev` and `pnpm alteration deploy` have different `pool`, that cause this problem.

I pushed a commit to fix that.

**Review by wangsijie — APPROVED** [review]

File: packages/schemas/alterations/next-1665300135-sign-in-sign-up.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/schemas/alterations/next-1665300135-sign-in-sign-up.ts:107]: Not related to this PR currently.
Discuss offline and create an issue when needed.

https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1666064801635719

**Pull request merged**